src/main/resources/openapi/courthouse.yaml

```diff
@@ -186,6 +186,8 @@ components:
               items:
                 type: integer
               example: [1, 2]
+            has_data:
+              type: boolean
```

Comment:
As the field name is vague, consider adding a `description` here. Something along the lines of "Indicates whether the courthouse has related cases or hearings"